mdetsims/sim_utils.py

```diff
@@ -165,13 +174,21 @@ def __init__(
             interpolation_type='cubic',
             ngal_factor=None):
         self.rng = rng
+        self.sel = sel
+        self.fraction = fraction
+        if self.fraction == 1:
+        	self.flag=False
+        else:
+        	self.flag=True
```

Comment:
We should use something more descriptive here too, maybe `self.flag_ex`?

Reply:
Yes, very precise！